js/collections/UsersShort.js

```diff
@@ -0,0 +1,31 @@
+/* use this collection for any list of users that can be fetched as an array of guids */
+
+import { Collection } from 'backbone';
+import UserShort from '../models/UserShort';
+import app from '../app';
+
+
+module.exports = Collection.extend({
+  /* we have to use the older style for this collection, the ES6 style creates a bug where models
+  cannot be removed using their ids */
+
+  initialize: function (models, options) {  // eslint-disable-line object-shorthand
+    if (!options.type) {
+      throw new Error('You must provide a type to the collection');
+    }
+
+    this.url = app.getServerUrl(options.guid === app.profile.id || !options.guid ?
```

Comment:
Can we declare this as a 'url' function so it's evaluated every time the url is needed as opposed to on object instantiation?